test/package.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var path = require('path');
+var assert = require('yeoman-assert');
+var helpers = require('yeoman-generator').test;
+
+describe('node:package', function () {
+	before(function (done) {
+		helpers.run(path.join(__dirname, '../generators/package'))
+			.on('end', done);
+	});
+
+	it('creates package.json', function () {
+		assert.file([
```

Comment:
we need better test here. Let's read and parse the package.json file and make sure the field are correct.